localstack/config.py

```diff
@@ -1286,7 +1283,6 @@ def use_custom_dns():
     "SYNCHRONOUS_SNS_EVENTS",
     "TEST_AWS_ACCOUNT_ID",
     "TF_COMPAT_MODE",
-    "USE_SINGLE_REGION",
```

Comment:
I think we should still forward the variable in order to make sure that we show the deprecation warnings / inform the user that it's not having any effect anymore.

Reply:
Good idea, fixed in 46efde0a7